src/callgraph_mode.ml

```diff
@@ -20,13 +24,13 @@ let param =
     (optional (Command.Arg_type.create of_string))
     ~doc:
       " When magic-trace is running with sampling collection mode, this sets how it \
-       should reconstruct callstacks. The options are Last_branch_record (lbr), Dwarf \
-       (dwarf), and Frame_pointers (fp). Will default to Last_branch_record if supported \
-       and Dwarf otherwise."
+       should reconstruct callstacks. The options are lbr/lbr-no-stitch/dwarf/fp or a \
+       sexp. Will default to lbr is available and dwarf otherwise. For more info: \
+       https://magic-trace.org/w/c"
 ;;
 
 let to_perf_string = function
```

Comment:
Hmm, maybe this should be of the form "to perf args", and contain the stiched lbr bits too.

Reply:
Perhaps, the only odd thing is args are needed for perf script and for perf record. Maybe two functions would be better then?

Reply:
I committed how I would do this if you want to look